src/credentials/attestation_verifier/DeviceAttestationVerifier.cpp

```diff
@@ -77,6 +81,25 @@ DeviceAttestationVerifier * gDacVerifier = &gDefaultDACVerifier;
 
 } // namespace
 
+#ifdef MATTER_FILE_SYSTEM_ENABLED
+FileAttestationTrustStore::FileAttestationTrustStore(const char * paaTrustStorePath)
+{
+    uint32_t certificateLength = 0;
+    std::ifstream ifs;
+
+    ifs.open(paaTrustStorePath, std::ifstream::in);
+
+    while (ifs.read(reinterpret_cast<char *>(&certificateLength), sizeof(uint32_t)))
```

Comment:
Let's use a glob pattern for all files matching, rather than relying on a "meta" list file, which has to be further explained.